### PR TITLE
Fix Icon Inconsistent colors and class names

### PR DESCRIPTION
**Pull request by coliff**
## Description

Partial fix for #1889

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Locally with Edge v120 on Windows 11.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/src/components/icons/icon-arrow-down.tsx [hunk 1]
```diff
@@ -12,13 +12,12 @@ interface IconProps {
...
   <svg
     class="icon-arrow-down"
-    xmlns="http://www.w3.org/2000/svg"
     fill={props.color ?? 'currentColor'}
     height={props.size ?? 16}
-    width={props.size ?? 16}
     onClick={props.onClick}
-    data-test-id="iconArrowDown"
-    viewBox="0 0 24 24">
+    viewBox="0 0 24 24"
```
Comment (msankaran0712) [inline stencil-workspace/src/components/icons/icon-arrow-down.tsx:18]: is data-test-id removed not breaking any unit test?

File: stencil-workspace/src/components/icons/icon-arrow-up.tsx [hunk 2]
```diff
@@ -12,13 +12,12 @@ interface IconProps {
 export const IconArrowUp: FunctionalComponent<IconProps> = (props: IconProps) => (
   <svg
     class="icon-arrow-up"
-    xmlns="http://www.w3.org/2000/svg"
     fill={props.color ?? 'currentColor'}
     height={props.size ?? 16}
-    width={props.size ?? 16}
     onClick={props.onClick}
-    data-test-id="iconArrowUp"
```
Comment (msankaran0712) [inline stencil-workspace/src/components/icons/icon-arrow-up.tsx:20]: data-test-id removed

File: stencil-workspace/src/components/icons/icon-info.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ export const IconInfo: FunctionalComponent<IconProps> = (props: IconProps) => (
       fill-rule="evenodd"
       clip-rule="evenodd"
       d="M12 21C16.968 21 21 16.968 21 12C21 7.032 16.968 3 12 3C7.032 3 3 7.032 3 12C3 16.968 7.032 21 12 21ZM11 7H13V9H11V7ZM11 11H13L13 17H11L11 11Z"
-      fill={props.color ?? '#6A6976'}
+      fill={props.color ?? 'currentColor'}
```
Comment (msankaran0712) [inline stencil-workspace/src/components/icons/icon-info.tsx:23]: could we not move this to svg?

File: stencil-workspace/src/components/icons/icon-arrow-down.tsx (hunk 1 above)
Comment (coliff) [reply stencil-workspace/src/components/icons/icon-arrow-down.tsx:18]: yep, I had to put it back

File: stencil-workspace/src/components/icons/icon-arrow-up.tsx (hunk 2 above)
Comment (coliff) [reply stencil-workspace/src/components/icons/icon-arrow-up.tsx:20]: put it back

**Review by msankaran0712 — APPROVED** [review]

**Pull request merged**